Src/WebUI/ClientApp/src/api-authorization/login/login.component.html

```diff
@@ -36,10 +29,3 @@ <h4>Use a local account to log in.</h4>
     </section>
   </div>
 </div>
```

Comment:
I see the toast component is removed.  How are we showing login messages to the user?

Reply:
Toast will come from `ToastrService` which will spawn a component instead - it has a nice built in simple component like success or error or warning so we can just use them if we want to just show a simple message 😄 

e.g.
```ts
this.toastrService.error('⚠️ Login failed. Please check your credentials and try again.');
```